AFL Player Cards_2020-09-29.sql

```diff
@@ -0,0 +1,69 @@
+# ************************************************************
```

Comment:
Try to avoid spaces in file names as different technology handles whitespace differently, there's no need to change it now. But on the command line you'd have to type AFL\ Player\ Cards to access this file.